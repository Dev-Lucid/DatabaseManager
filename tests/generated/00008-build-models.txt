df3fb038f2f1097ddf127fbdf25b709e
0f19eee7c1b3a384d2b85be11ee50060
c7808cd8398c65d43d4a1c765b57f1e2
01229f904b5c5a69649f76ecc9137bd9
2b9a64f868af59d16ae04d52010d491c
e21baafc90c9986e9e2dbb031d18abe0
5cda48e32c803a064f1e029a70f707e5
e109ad4403b46854ba95139fdd3a1b3a
