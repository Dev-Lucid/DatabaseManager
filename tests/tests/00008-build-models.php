<?php
global $__dbm;
$cmd = 'php -f '.__DIR__.'/../../bin/build_models.php ';
$cmd .= ' type='.$__dbm['type'];
$cmd .= ' database='.$__dbm['database'];
$cmd .= ' username='.$__dbm['username'];
$cmd .= ' password='.$__dbm['password'];
$cmd .= ' host='.$__dbm['host'];
$cmd .= ' model_path='.$__dbm['model_path'];
shell_exec($cmd);

$out = '';

$files = glob($__dbm['model_path'].'/*'); // get all file names
foreach($files as $file)
{
	
	if(is_file($file) && !is_dir($file))
	{
		#echo('hashing: '.$file."\n");
		$out .= md5(file_get_contents($file))."\n";
	}
}

$files = glob($__dbm['model_path'].$__dbm['base_subdir'].'/*'); // get all file names
foreach($files as $file)
{ 
	if(is_file($file) && !is_dir($file))
	{
		#echo('hashing: '.$file."\n");		
		$out .= md5(file_get_contents($file))."\n";
	}
}


file_put_contents($output_path,$out);
?>