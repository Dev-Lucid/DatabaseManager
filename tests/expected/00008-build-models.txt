c7808cd8398c65d43d4a1c765b57f1e2
01229f904b5c5a69649f76ecc9137bd9
5cda48e32c803a064f1e029a70f707e5
e109ad4403b46854ba95139fdd3a1b3a
